### PR TITLE
Update browser and devices lists

### PR DESCRIPTION
**Pull request by PraveenAsokan**
- [x] Brower list and devices are updated in the docs

**Ticket** - https://github.com/ConnectedHomes/nucleus/issues/2220

### CODE REVIEW

File: docs/getting-started/working-with-nucleus.md [hunk 1]
```diff
@@ -39,30 +39,25 @@ There are many ways to access www.britishgas.co.uk and we strive to ensure it's
...
 | :--- | :--- | :--- | :--- |
-| Computer | Windows 10 | Chrome | 83 |
-| Computer | Windows 10 | Edge | 83 |
-| Computer | Windows 10 | Internet Explorer | 11 |
-| Computer | Windows 10 | Firefox | 77 |
-| Computer | Windows 10 | Edge | 18 |
+| Computer | Windows 10 | Chrome | 90 |
+| Computer | Windows 10 | Firefox | 83 |
+| Computer | Windows 10 | Edge | 85 |
 | Computer | MacOS Catalania | Safari | 13.1 |
```
Comment (PtitBen) [inline docs/getting-started/working-with-nucleus.md:49]: MacOS Catalina

File: docs/getting-started/working-with-nucleus.md [hunk 2]
```diff
@@ -39,30 +39,25 @@ There are many ways to access www.britishgas.co.uk and we strive to ensure it's
...
-| Computer | MacOS Catalania | Chrome | 83 |
-| iPhone | iOS 12 (iPhone 7) | Safari |13 |
+| Computer | MacOS Catalania | Chrome | 88 |
```
Comment (PtitBen) [inline docs/getting-started/working-with-nucleus.md:50]: MacOS Catalina

File: docs/getting-started/working-with-nucleus.md [hunk 3]
```diff
@@ -39,30 +39,25 @@ There are many ways to access www.britishgas.co.uk and we strive to ensure it's
...
-| iPad | iPad OS 13 (iPad 7th) | Safari | 13 |
-| iPad | iPad OS 13 (iPad 7th) | Chrome | 80 |
+| iPhone | iOS 14 (iPhone 12) | Chrome | 86 |
+| iPhone | iOS 13 (iPhone XS) | Chrome | 86 |
+| iPad | iPad OS 14 (iPad Pro 12.9 2020) | Safari | 13 |
+| iPad | iPad OS 14 (iPad Pro 12.9 2020) | Chrome | 86 |
+| Android | Android OS 10 (Samsung Galaxy S20) | Chrome | 89 |
+| Android | Android OS 11 (Samsung Galaxy S21) | Samsung Browser | 11.1 |
+
+**Note :** IE11 and EDGE 18 has been deprecated from the list as the usage of these browsers are very minimal and we stopped supporting IE11 eventually.
```
Comment (PtitBen) [inline docs/getting-started/working-with-nucleus.md:60]: IE11 and Edge 18 have been deprecated from this list as the usage levels for these browsers are very minimal (around 0.5% as of June 2021).

Comment (dbenmore) [reply docs/getting-started/working-with-nucleus.md:60]: IE11 and Edge 18 browsers are no longer supported as their usage levels are below 0.5% (as of June 2021).

Comment (PraveenAsokan) [reply docs/getting-started/working-with-nucleus.md:60]: @Ben - Do you want me to remove this `we stopped supporting IE11 eventually.` from the sentence

Comment (PtitBen) [reply docs/getting-started/working-with-nucleus.md:60]: Use Dan's and yes remove the last part :)

**Review by PtitBen — APPROVED** [review]
All looking good!

**Pull request merged**